dom/fifty-shades-of-cold_test.js

```diff
@@ -13,10 +13,13 @@ export const tests = []
 tests.push(async ({ page, eq }) => {
   // check that the css is properly generated
 
-  const style = await page.$$eval('style', nodes => nodes[1].innerHTML)
-  const classes = style
-    .split('}')
-    .map(s => s.replace(/(\.|{|:|;|\s+)/g, ''))
+  const style = await page.$$eval('head style', nodes =>
+    nodes.map(n => n.innerText),
+  )
+
+  const classes = style[0]
+    .split('\n')
```

Comment:
This line will make valid solutions fail the test since start a new line for every new element is a best practice but has no semantic value in CSS.

It could be replaced by `.split('}')`